src/Microsoft.DocAsCode.Build.ManagedReference/ManagedReferenceDocumentProcessor.cs

```diff
@@ -145,7 +145,8 @@ public override SaveResult Save(FileModel model)
                 UidLinkSources = model.UidLinkSources,
                 XRefSpecs = (from item in vm.Items
                              from xref in GetXRefInfo(item, model.Key)
-                             select xref).ToImmutableArray(),
+                             group xref by xref.Uid into g
+                             select g.First()).ToImmutableArray(),
```

Comment:
>select g.First()) [](start = 29, length = 17)

is there any info of overload item that couldn't be auto-generated but could be provided by yaml file? namely, should we do a merge if duplicate?